include/swift/IDE/CodeCompletion.h

```diff
@@ -705,17 +704,18 @@ class CodeCompletionResult {
                        StringRef BriefDocComment,
                        ArrayRef<StringRef> AssociatedUSRs,
                        ArrayRef<std::pair<StringRef, StringRef>> DocWords,
+                       enum ExpectedTypeRelation TypeDistance,
```

Comment:
Why is "enum" used here?  I would expect the type name to be unambiiguous in this context

Reply:
Oops, just a copy/paste mistake.